try {
} catch (error) {
  // error.message
  (error as Error).message;
}
